app/scripts/modules/cloudfoundry/src/serverGroup/details/sections/cloudFoundryInfoDetailsSection.tsx

```diff
@@ -75,12 +75,15 @@ export class CloudFoundryInfoDetailsSection extends React.Component<ICloudFoundr
             <dd>{serverGroup.memory}</dd>
           </dl>
         </CollapsibleSection>
-        <CollapsibleSection heading="Package" defaultExpanded={true}>
-          <dl className="dl-horizontal dl-flex">
-            <dt>Checksum</dt>
-            <dd>{serverGroup.droplet.packageChecksum}</dd>
```

Comment:
Yes we can remove it from the `ICloudFoundryDroplet`

There doesn't seem to be any references anymore to `packageChecksum` in the clouddriver model `CloudFoundryDroplet`

yes there are differences between a droplet checksum and a package checksum. package checksum would refer to the checksum of the code that is executing. The checksums between the droplet and package could differ because the droplet could be built using different buildpacks.